third-upgrades/content/config/creatures/dungeon/lvl6Upg2.json

```diff
@@ -104,15 +104,15 @@
 				"bonus" :
 				{
 					"type" : "SPELL_AFTER_ATTACK",
-					"subtype" : 81 // acidBreathDamage
+					"subtype" : "spell.acidBreath"
```

Comment:
I guess, addInfo in this code has deafult values, no need to add this syntax (?)

Reply:
No idea what the intention is with this ability. What level for example? There is no warning ATM so I leave it for future modders to change.

Reply:
Like all creature-origin spells, they have 'no-level', so I'm almost sure this syntax is enough in such cases.